lib/Echidna/UI.hs

```diff
@@ -112,11 +111,11 @@ monitor cleanup = let
 
 -- | Heuristic check that we're in a sensible terminal (not a pipe)
 isTerminal :: MonadIO m => m Bool
-isTerminal = liftIO $ queryTerminal (Fd 0)
+isTerminal = liftIO $ (&&) <$> queryTerminal (Fd 0) <*> queryTerminal (Fd 1)
```

Comment:
when do we need this check?

Reply:
piping to another program or file will still cause the ui to run, e.g. `stack run echidna-test -- solidity.sol --config cfg 1>output.log 2>/dev/null` and users will need to `C-c` out of echidna without seeing the UI.